schedule/yam/agama_unattended.yaml

```diff
@@ -1,7 +1,7 @@
 ---
-name: agama_auto
+name: agama_unattended
 description: >
-  Prepare url for agama.auto boot parameter, boot, perform auto-installation with agama.
+  Prepare url for agama.auto boot parameter, boot, perform auto-installation with agama and publsih qcow.
```

Comment:
there is no more agama.auto...let's rephrase this to:
"Default unattended installation" (publish qcow2 are technical things...)

Reply:
updated